.travis.yml

```
@@ -0,0 +1,21 @@
+language: scala
+
+jdk: openjdk8
+
+scala:
+  - 2.13.5
+
+services:
+  - postgresql
+
+before_script:
+  - psql -c 'create database dummyproject;' -U postgres
+  - psql -c 'create table userdb(
+    id uuid PRIMARY KEY NOT NULL,
+    username VARCHAR UNIQUE NOT NULL,
+    name     VARCHAR NOT NULL
+    );' -U postgres
+
+script:
+  - sbt ++$TRAVIS_SCALA_VERSION test
```

Based on the suggestion about the config, we could rewrite this as:
```suggestion
  - psql -c 'create database dummyproject;' -U postgres
  - psql -c 'create table userdb(
    id uuid PRIMARY KEY NOT NULL,
    username VARCHAR UNIQUE NOT NULL,
    name     VARCHAR NOT NULL
    );' -U postgres

script:
  - env DB_URL=jdbc:postgresql:dummyproject DB_USERNAME=postgres sbt ++$TRAVIS_SCALA_VERSION test
```